### DISCUSSION TITLE
疫情期間當阿宅，17 歲少年做出意念控制的機械手臂

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#631] [General]
為什麼別人17歲那麼強 我都不知道我17歲的時候在幹什麼
https://technews.tw/2022/06/04/17-year-old-kid-genius-built-a-mind-controlled-prosthetic-arm-in-his-spare-time/

**Comment by magicmicheal** [comment]
最近剛踢完世足，看了許多球員介紹，確實常有標題的這種感覺了，大部分的球星在小時候就才華洋溢，甚至有少數天才快要成年才參與訓練，還能成為那歌方面的佼佼者，其實不管是運動，大部分的學科也是，儘管大家都說努力很重要，但數學相關的教授想必認為天分是不可或缺的，努力確實能讓工數考高分，但想要找到新的公式必然需要一些天分

**Comment by awehao** [comment]
我曾說過，人生在世，要找到自己最有興趣的事，想必這位少年很早就找到自己的興趣，並努力學習，雖然我現在仍然渾渾噩噩，但也有在不斷嘗試，希望有朝一日，也能成為自己喜歡的樣子，現階段要盡好學生的本分，奮發向上才行，有天分固然重要，但後天的努力才是重點

**Reply by chia-kai-chang** [reply]
跟大家分享一下，這些優美的文字是怎麼產生的，應該很有意思~

**Reply by awehao** [reply]
........我自己打的老師，其實也沒甚麼意思，只是表達我的想法僅此而已，這些優美的文字怎麼產生的，想想我也活快快20年了老師，可以自己打出來

**Reply by magicmicheal** [reply]
我認為 AI可能比我用了20年的大腦優美，不過我認為他至少有切合主題，比一些更明顯的唬爛模擬器還認真

**Reply by awehao** [reply]
謝謝同學，只能說AI的天分比我們好啦，我只是盡我所能表達自己的想法，唬爛模擬器這種東西，我是不會用啦，看老師如何解讀啦，只能說，"功力不夠，學無止境"，AI的發明，讓人與人之間產生了誤解，看來是雙面刃呢

**Comment by awkward-willy** [comment]
透過儀器偵測腦電波、神經訊號等生理數值，並透過各式各樣的儀器展現出來真的很驚人。或許未來我們真的能單純透過腦中的想法去操控各項科技，進而手機、穿戴式裝置的需求，達到真正靠「意念」就可以做到各項事情。

**Reply by chia-kai-chang** [reply]
我對這個領域也蠻感興趣的

歡迎同學之後來Ai專題跟我一起做

**Comment by RogelioKG** [comment]
好猛，沒想到是使用非侵入式裝置就達成操控，
照理來說這樣精準度應該有所下降才對。
應該主要是仰賴頭部姿勢的判定吧 (不過這樣也是很猛了)

**Comment by manipulateed** [comment]
透過這項技術真的能方便很多身障人士的生活，我也挺好奇他到底怎麼做出來的，肯頂也是經過很多學習以及矜持下去的精神吧，未來有機會我也想要嘗試這個領域。

**Comment by larry40305** [comment]
如果真的能實現意念控制，那世界的肥胖指數絕對會上升。

**Comment by 0212Jerry** [comment]
之前看過馬斯克的Neurolink。也是在做類似的東西，利用植入腦部的晶片控制螢幕上的物體或機械手臂，當初看完也覺得很神奇。不過現在看到一個十七歲的人做著看似科學家在做的事情，不由得佩服。

**Comment by Vincent20030314** [comment]
有了意念控制的機械手臂隨然很方便，但是也會讓人類懶惰不少。
